sdk/core/Azure.Core.Amqp/src/Azure.Core.Amqp.csproj

```diff
@@ -0,0 +1,19 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Are we expecting to have extra dependencies in this package?

Reply:
Hmm most likely not. why?

Reply:
Why split the package then?

Reply:
Ah, this was the recommendation from @KrzysztofCwalina. He wanted a separate dll for this functionality.

Reply:
To get what benefit?